aggregator/src/main/java/com/redhat/cloud/notifications/db/EmailAggregationResources.java

```diff
@@ -24,7 +25,7 @@
                 .getResultList();
     }
 
-    public List<EmailAggregation> getEmailAggregation(EmailAggregationKey key, LocalDateTime start, LocalDateTime end) {
+    List<EmailAggregation> getEmailAggregation(EmailAggregationKey key, LocalDateTime start, LocalDateTime end) {
```

Comment:
This method is only used in an Integration Test. We should refactor it in a future PR.